src/ui/public/courier/__tests__/saved_object.js

```diff
@@ -88,11 +88,11 @@ describe('Saved Object', function () {
     // The default implementation of safeConfirm uses $timeout which will cause
     // the test environment to hang.
     function ($provide) {
-      $provide.decorator('safeConfirm', () => {
-        return (message) => {
+      $provide.decorator('safeConfirm', () =>
```

Comment:
ah right, two implicit returns. :) yea, this is getting confusing, opted for the second version.